server/src/main/java/org/elasticsearch/index/query/SearchExecutionContext.java

```diff
@@ -635,8 +635,12 @@ public Index getFullyQualifiedIndex() {
         if (runtimeMappings.isEmpty()) {
             return Collections.emptyMap();
         }
-        Map<String, RuntimeField> runtimeFields = RuntimeField.parseRuntimeFields(new HashMap<>(runtimeMappings),
-            mapperService.parserContext(), false);
+        //TODO add specific tests to SearchExecutionTests similar to the ones in FieldTypeLookupTests
```

Comment:
Checking I'm not missing something, this is not really related to the change?

Reply:
No, I think this was just a reminder that Luca added because test coverage here isn't great